cypress/e2e/API/EngageSphere.cy.js

```diff
@@ -0,0 +1,151 @@
+/// <reference types="Cypress" />
+describe('Validações de API', () => {
+    context('Solicitações inválidas de forma elegante', () => {
+      it('Então valido page="invalid" como string', () => {
+        cy.request({
+          method: 'GET',
+          url: 'http://localhost:3001/customers?page=invalid&limit=10&size=All',
+          failOnStatusCode: false 
+        }).then(response => {
```

Comment:
Que tal usar desestruturação de objetos, como abaixo?

```suggestion
        }).then(({ status })=> {
```